docs/faq.md

```diff
@@ -18,7 +18,9 @@ CMake Tools was created separately from the [CMake extension](https://marketplac
 
 CMake Tools is not the same as CMake. There are many great resources around to learn how to use CMake. See Jason Turner's [C++ Weekly - Intro to CMake](https://www.youtube.com/watch?v=HPMvU64RUTY) for a good video introduction.
 
-[CMake's documentation](https://marketplace.visualstudio.com/items?itemName=twxs.cmake) is also available.
+[CMake's documentation](https://cmake.org/cmake/help/latest/) is also available.
+
+One of the maintainers of CMake, Craig Scott, publishes a (paid) book, ["Professional CMake"](https://crascit.com/professional-cmake/).
```

Comment:
These resources might read better as a bulleted list.  Could you change it to something more along these lines?
```
CMake Tools is not the same as CMake. There are many great resources around to learn how to use CMake:
* Jason Turner's [C++ Weekly - Intro to CMake](https://www.youtube.com/watch?v=HPMvU64RUTY) is a good video introduction.
* One of the maintainers of CMake, Craig Scott, publishes a (paid) book, ["Professional CMake"](https://crascit.com/professional-cmake/)
* [CMake's documentation](https://cmake.org/cmake/help/latest/) is also available.
```

You can also add a line to CHANGELOG.md to give yourself credit for your contribution.  Thank you!

Reply:
updated :)